braga/README.md

```diff
@@ -0,0 +1,655 @@
++ Comparatively ***high education levels*** and low crime rate
++ Considerably better **air quality** than most major European Cities
++ Parks, Out-door Gyms, Mountain Biking Trails
++ Excellent transport links (_Buses/Trains to the rest of Portugal_)
+  > _Direct_ Bus to Porto Airport is **€8** and takes **50 mins**
+  via [GetBus](https://getbus.eu/en/braga-airport-braga) <br />
+  > Bus stop _in front_ of the house: [Braga House Street View](https://www.google.pt/maps/@41.547696,-8.4145142,3a,75y,90h,90t/data=!3m6!1e1!3m4!1s1nlC6YYTE91Q2njoTZhVmg!2e0!7i13312!8i6656)
+
+## _Who?_
+
+The _objective_ of `Braga House` is _not_
+to create a "Hostel" for the "_general public_". <br />
+The _objective_ is to surround ourselves with people who are
+interesting, creative, optimistic and ambitious! <br />
+This criteria _automatically exculdes_ 95% of the population
+who prefer to sit around watching TV and _complain_ <br />
+that there are "***no jobs***";
+when in _fact_ there's a
+["_skills gap_"](https://hbr.org/2014/08/employers-arent-just-whining-the-skills-gap-is-real)
+i.e: ***huge shortage*** of
+people with specific technical and creative skills.
+We want to _attract_ people who are pursuing/practicing these skills
+and help them _flourish_!
+
+
+<!-- ![dont-need-an-alarm-clock](https://cloud.githubusercontent.com/assets/194400/25293342/63989f6e-26d2-11e7-938c-576b0f2f86c1.png) -->
+
+`Braga House` will be available to **two groups** of people:
+
+### Residents (_Long Term_)
+
+People who (_temporarily_) ***live*** in the house.
+The _amount_ of time a person will _live_ in `Braga House` will
+range from **1-6 Months**
+(_we don't `expect` people to stay longer than
+a few months but we won't forbid anyone
+from staying for as long as they like!_)
+
+We will give preferential residence/access to people who (_already_)
+work with/for dwyl.
+
+### Guests (_Short Term_)
+
+Guests are people who are visiting for a short period (1 night - 1 week).
+We _love_ having guests because they bring new ideas/perspective
+and tell us how we can improve.
+
+Guests will need to fill in a brief "_application_" form
+to come and stay with us.
+(_we aren't focussing on guests for the first 6 months,
+guest form will only be created later._)
+
+Guests will be able to work their choice of creative/technology projects,
+collaborate with other people in the house on one of the existing projects
+or simply relax and absorb the atmosphere.
+
+
+
+# How?
+
+
+## The House
+
+The house is comprised of 3 floors:
++ **Basement**
+  + Large Garage with street door and ramp to the public road - we will
+  be using the garage as our multi-purpose indoor communal/community area.
+    + Projector
+    + Beanbags
+    + Yoga Mats
+  + 2 Medium Sized ("double") rooms:
+    + **Multi-media recording/editing studio** for video/podcast production
+    + "**Art Room**" for design, crafts & other arts.
+  + 1 Small ("single") room (_was previously used as a study_)
```

Comment:
The whole basement will be wheelchair accessible and we will be building a ramp from the garden level into the kitchen entrance so essentially only the top floor won't have wheelchair access.